eppo_core/src/events/event_dispatcher.rs

```diff
@@ -26,6 +27,12 @@ pub struct EventDispatcherConfig {
     pub max_retries: Option<u32>,
 }
 
+#[derive(Debug)]
```

Comment:
minor: take a look at `thiserror` (there are examples in eppo core). It helps with deriving boilerplate code, so we can make this type a proper rust error